package.json

```diff
@@ -43,7 +43,14 @@
       "undici@<=5.28.2": ">=5.28.3",
       "tar@<6.2.1": ">=6.2.1",
       "braces@<3.0.3": ">=3.0.3",
-      "ws@<8.17.1": ">=8.17.1"
+      "ws@<8.17.1": ">=8.17.1",
+      "app-builder-lib@<24.13.2": ">=24.13.2",
+      "follow-redirects@<=1.15.5": ">=1.15.6",
+      "vite@>=4.0.0 <=4.5.2": ">=4.5.3",
+      "undici@>=6.0.0 <6.11.1": ">=6.11.1",
+      "ejs@<3.1.10": ">=3.1.10",
+      "electron-updater@<=6.3.0-alpha.5": ">=6.3.0-alpha.6",
+      "fast-loops@<1.1.4": ">=1.1.4"
```

Comment:
Whoa, that's a lot of new overrides, why are they needed?

Reply:
audit.